src/h5web/visualizations/heatmap/config.ts

```diff
@@ -1,11 +1,10 @@
 import type { ColorMap } from './models';
 import { Domain, ScaleType } from '../shared/models';
 import { StorageConfig, createPersistableState } from '../../storage-utils';
-import { findDomain } from '../shared/utils';
 
 interface HeatmapConfig {
   dataDomain: Domain | undefined;
-  initDataDomain: (values: number[]) => void;
+  resetDomains: (dataDomain: Domain | undefined) => void;
```

Comment:
Wasn't sure what to call it. I hope this works.

Reply:
`initDomains` ? `resetDomains` is fine to me tho.

Reply:
Yeah, I thought `initDomains` didn't convey the fact that the domains could change (it's not just called on page load or mount or whatever).